plugins/se.infomaker.publishflow/PublishFlowComponent.js

```diff
@@ -51,13 +52,9 @@ class PublishFlowComponent extends Component {
         })
 
         api.events.on(pluginId, event.DOCUMENT_CHANGED_EXTERNAL, (event) => {
-            if (event.data.key === 'pubStatus') {
-                this.extendState({
-                    status: api.newsItem.getPubStatus(),
-                    hasPublishedVersion: api.newsItem.getHasPublishedVersion(),
-                    pubStart: api.newsItem.getPubStart(),
-                    pubStop: api.newsItem.getPubStop(),
-                })
+            const change = getExternalChange(event)
+            if (change) {
+                this.extendState(change)
```

Comment:
What happens if `getExternalChange` returns null and extends the state with that?